src/storable.rs

```diff
@@ -492,6 +492,120 @@ where
     };
 }
 
+fn get_size_len<T>() -> usize
+where
+    T: Storable,
+{
+    match T::BOUND {
+        Bound::Bounded {
+            max_size,
+            is_fixed_size,
+        } => bytes_to_store_size(&Bounds {
+            max_size,
+            is_fixed_size,
+        }) as usize,
+        Bound::Unbounded => 8,
```

Comment:
(communication) why are we returning `8` here? As a reader this isn't clear to me.